### ISSUE TITLE
Fix buggy popovers on Rankings page

### ISSUE THREAD

**Issue by jorjo1** [Rostlab/JS16_ProjectF#103]
> Warning: Failed propType: The prop 'id' is required to make 'Popover' accessible for users using assistive technologies such as screen readers Check the render method of `Ranking`.warning @ (program):45
> (program):49 
> 
> Warning: [react-bootstrap] Specifying only the `"hover"` trigger limits the visibilty of the overlay to just mouse users. Consider also including the `"focus"` trigger so that touch and keyboard only users can see the overlay as well.warning @ (program):49
> 2browser.js?b253:49 
> 
> Warning: [react-bootstrap] Specifying only the `"hover"` trigger limits the visibilty of the overlay to just mouse users. Consider also including the `"focus"` trigger so that touch and keyboard only users can see the overlay as well.
